bin/dispatch-conf

```diff
@@ -434,6 +436,33 @@ class dispatch:
 
         perform_conf_update_session_hooks("post-session")
 
+    def copy_selinux_label(self, curconf, newconf):
+        """Copy the SELinux security label from the current config file to
+        the new/merged config file."""
+        try:
+            _ = os.listxattr(curconf)
```

Comment:
This call isn't really needed. I would check errno for ENOTSUP when you do the getxattr call instead.

Reply:
Good idea. Thanks!